crates/router/src/connector/noon/transformers.rs

```diff
@@ -73,11 +75,34 @@ pub struct NoonCard {
     cvv: Secret<String>,
 }
 
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct NoonApplePay {
+    payment_token: String,
+}
+
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct NoonGooglePay {
+    api_version_minor: u8,
+    api_version: u8,
+    payment_method_data: api_models::payments::GooglePayWalletData,
```

Comment:
all the fields inside `api_models::payments::GooglePayWalletData` would be serialized with snake_case, Does the connector also requires the same case?